examples/vacanza.py

```
@@ -0,0 +1,323 @@
+    Raises:
+        ValueError: If country code is not found in registry
+    """
+
+    # Check if it's in the COUNTRIES registry
+    for country, country_codes in COUNTRIES.items():
+        if (
+            country_code.lower() in country_codes
+            or country_code.upper() in country_codes
+            or country_code.title() in country_codes
+        ):
+            return COUNTRIES[country][1].upper()  # return upper case version
```

Ruff (0.15.10)

[warning] Unnecessary lookup of dictionary value by key

Use existing variable

(PLR1733)